Extension/tests/rule-constructor/test-rule-constructor.js

```diff
@@ -53,37 +53,37 @@ QUnit.test("Rules Constructor for DevTools", function(assert) {
     var ruleText = AdguardRulesConstructorLib.constructRuleText(element, options);
     assert.equal(ruleText, 'example.org###test-div');
 
-    options.isBlockSimilar = true;
-    options.classesSelector = '';
+    options.cssSelectorType = 'SIMILAR';
+    options.classList = [];
     ruleText = AdguardRulesConstructorLib.constructRuleText(element, options);
     assert.equal(ruleText, 'example.org');
 
-    options.classesSelector = null;
+    options.classList = null;
     options.excludeTagName = false;
     ruleText = AdguardRulesConstructorLib.constructRuleText(element, options);
-    assert.equal(ruleText, 'example.org##div.test-class.test-class-two');
+    assert.equal(ruleText, 'example.org##.test-class, .test-class-two');
```

Comment:
It's correct. When we are constructing css selector with type 'SIMILAR' (matching any element that contains any of classes) we should omit tag name.